js/sliderModel.js

```diff
@@ -70,84 +69,78 @@ define([
         return;
       }
 
-      var items = this.get('_items');
-      var userAnswer = this.get('_userAnswer');
-      for (var i = 0, l = items.length; i < l; i++) {
-        var item = items[i];
-        if (item.value === userAnswer) {
-          item.selected = true;
-          this.set('_selectedItem', item);
-          break;
-        }
+      const items = this.get('_items');
+      const userAnswer = this.get('_userAnswer');
+      const selectedItem = items.find(({ value }) => value === userAnswer);
+      if (selectedItem) {
+        selectedItem.selected = true;
+        this.set('_selectedItem', selectedItem);
       }
 
       this.setQuestionAsSubmitted();
       this.markQuestion();
       this.setScore();
       this.setupFeedback();
-    },
+    }
 
     //This preserves the state of the users answers for returning or showing the users answer
-    storeUserAnswer: function() {
+    storeUserAnswer() {
       this.set('_userAnswer', this.get('_selectedItem').value);
-    },
+    }
 
-    resetUserAnswer: function() {
+    resetUserAnswer() {
       this.set({
         _isAtLeastOneCorrectSelection: false,
         _userAnswer: undefined
       });
       this.selectDefaultItem();
-    },
-
-    deselectAllItems: function() {
-      _.each(this.get('_items'), function(item) {
-        item.selected = false;
-      }, this);
-    },
-
-    isCorrect: function() {
-      var numberOfCorrectAnswers = 0;
+    }
 
-      _.each(this.get('_items'), function(item, index) {
+    deselectAllItems() {
+      this.get('_items').forEach(item => item.selected = false);
+    }
+    
+    isCorrect() {
+      let numberOfCorrectAnswers = 0;
+      this.get('_items').forEach(item => {
```

Comment:
```suggestion
      this.set('_isAtLeastOneCorrectSelection', false);
      this.get('_items').forEach(item => {
```

Reply:
Part 1 to remove the weird `return this.get('_isAtLeastOneCorrectSelection') ? true : false;` ternary

Reply:
Using a filter would make things a lot simpler:
```js
isCorrect() {
  const numberOfCorrectAnswers = this.get('_items').filter(({ selected, correct }) => selected && correct).length;

  this.set('_isAtLeastOneCorrectSelection', numberOfCorrectAnswers > 0);
  this.set('_numberOfCorrectAnswers', numberOfCorrectAnswers);

  return this.get('_isAtLeastOneCorrectSelection');
}
```